depends/install_extra_test_images.sh

```diff
@@ -1,15 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
For the record, what's the thinking behind this change?

For some reason, `depends/install_raqm*.sh` has `#!/usr/bin/env bash`, but the other `depends/install_*.sh` scripts have `#!/bin/bash`.

Reply:
For portability:

https://stackoverflow.com/a/10383546/724176

Probably doesn't matter too much, but this file was being updated anyway.